service/src/test/resources/dynamicConfig.yaml

```diff
@@ -33,4 +33,13 @@
      celeborn.test.tenant.timeoutMs.only: 100s
      celeborn.test.tenant.enabled.only: false
      celeborn.test.tenant.int.only: 10
+     celeborn.client.push.queue.capacity: 1024
+
+-  tenantId: tenant_id
```

Comment:
The dynamic config of user in yaml could be:

```
-  tenantId: tenant_id
   level: TENANT
   config:
     celeborn.client.push.buffer.initial.size: 10k
     celeborn.client.push.buffer.initial.size.only: 100k
     celeborn.worker.fetch.heartbeat.enabled: false
     celeborn.test.tenant.timeoutMs.only: 100s
     celeborn.test.tenant.enabled.only: false
     celeborn.test.tenant.int.only: 10
     celeborn.client.push.queue.capacity: 1024
   users: 
     - name: AngersZhuuu
       config:
         celeborn.client.push.buffer.initial.size: 10k
         celeborn.client.push.buffer.initial.size.only: 100k
         celeborn.worker.fetch.heartbeat.enabled: false
         celeborn.test.tenant.timeoutMs.only: 100s
         celeborn.test.tenant.enabled.only: false
         celeborn.test.tenant.int.only: 10
         celeborn.client.push.queue.capacity: 1024
     - name: PureBlue
       config:
         celeborn.client.push.buffer.initial.size: 20k
         celeborn.client.push.buffer.initial.size.only: 200k
         celeborn.worker.fetch.heartbeat.enabled: true
         celeborn.test.tenant.timeoutMs.only: 200s
         celeborn.test.tenant.enabled.only: true
         celeborn.test.tenant.int.only: 20
```

Reply:
> The dynamic config of user in yaml could be:
> 
> ```
> -  tenantId: tenant_id
>    level: TENANT
>    config:
>      celeborn.client.push.buffer.initial.size: 10k
>      celeborn.client.push.buffer.initial.size.only: 100k
>      celeborn.worker.fetch.heartbeat.enabled: false
>      celeborn.test.tenant.timeoutMs.only: 100s
>      celeborn.test.tenant.enabled.only: false
>      celeborn.test.tenant.int.only: 10
>      celeborn.client.push.queue.capacity: 1024
>    users: 
>      - name: AngersZhuuu
>        config:
>          celeborn.client.push.buffer.initial.size: 10k
>          celeborn.client.push.buffer.initial.size.only: 100k
>          celeborn.worker.fetch.heartbeat.enabled: false
>          celeborn.test.tenant.timeoutMs.only: 100s
>          celeborn.test.tenant.enabled.only: false
>          celeborn.test.tenant.int.only: 10
>          celeborn.client.push.queue.capacity: 1024
>      - name: PureBlue
>        config:
>          celeborn.client.push.buffer.initial.size: 20k
>          celeborn.client.push.buffer.initial.size.only: 200k
>          celeborn.worker.fetch.heartbeat.enabled: true
>          celeborn.test.tenant.timeoutMs.only: 200s
>          celeborn.test.tenant.enabled.only: true
>          celeborn.test.tenant.int.only: 20
> ```

The yaml structure looks good to me, @AngersZhuuuu

Reply:
maybe `tenantId` => `tenant` for consistent?

Reply:
Done